core/src/signals.rs

```diff
@@ -21,11 +21,11 @@ impl<T: Shutdown + Send + Sync> Signals<T> {
         ctrlc::set_handler(move || {
             let v = signals.iterations.fetch_add(1, Ordering::SeqCst);
             if v > 1 {
-                println!("^SIGNAL - halting");
+                println!("^SIGTERM - halting");
                 std::process::exit(1);
             }
 
-            println!("^SIGNAL - shutting down... (CTRL+C again to halt)");
```

Comment:
As the Kaspad output is displayed by KOS (and perhaps later by CLI) it becomes confusing as user is told about Ctrl+C but that doesn't actually work (or if daemon support is added to CLI it will actually cause CLI to shutdown).

I've contemplated adding a special command line key to Kaspad..  but meanwhile has removed the message to prevent possible confusion.